website/src/styles/global.css

```
@@ -6,9 +14,56 @@
 
 @theme {
   --font-mono:
-    ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier New',
-    monospace;
-  --font-sans: ui-sans-serif, -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
+    'JetBrains Mono Variable', ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas,
```

Stylelint (17.14.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)